plugins/slidev/skills/create-slide/references/setup-guide.md

```
@@ -558,190 +558,190 @@ surge
 ```bash
-# Monaco 관련 패키지 확인
+# check the Monaco-related package
 npm install monaco-editor@latest
 ```
 
-#### 6. PDF/PNG 내보내기 실패
+#### 6. PDF/PNG export failure
 
-**증상:**
+**Symptom:**
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)